Sources/Testing/ABI/EntryPoints/EntryPoint.swift

```diff
@@ -509,9 +510,9 @@ public func configurationForEntryPoint(from args: __CommandLineArguments_v0) thr
   }
 
   // Attachment output.
-  if let attachmentsPath = args.experimentalAttachmentsPath {
+  if let attachmentsPath = args.attachmentsPath {
     guard fileExists(atPath: attachmentsPath) else {
```

Comment:
Tangential this PR and can be fixed separately, but I notice this throws an error if the attachment directory doesn't exist yet. I find it helpful when tools create directories like this on the user's behalf if it doesn't exist yet.

Reply:
I want to discuss the exact behaviour of the flag with the SwiftPM folks first; for now, this is the behaviour we already implemented.